js/main.js

```diff
@@ -0,0 +1,30 @@
+const lBars1 = document.querySelector('.line1__bars-menu');
+const lBars2 = document.querySelector('.line2__bars-menu');
+const lBars3 = document.querySelector('.line3__bars-menu');
+const listBars = document.querySelector('.list_bar-menu');
+
+document.querySelector('#bmenu').addEventListener('click', () => {
+  lBars1.classList.toggle('activeline1__bars-menu');
+  lBars2.classList.toggle('activeline2__bars-menu');
+  lBars3.classList.toggle('activeline3__bars-menu');
+  listBars.toggle('deploy');
+});
+document.querySelector('.a0').addEventListener('click', () => {
+  lBars1.classList.toggle('activeline1__bars-menu');
+  lBars2.classList.toggle('activeline2__bars-menu');
+  lBars3.classList.toggle('activeline3__bars-menu');
+  listBars.toggle('deploy');
+});
+
+document.querySelector('.a1').addEventListener('click', () => {
+  lBars1.classList.toggle('activeline1__bars-menu');
+  lBars2.classList.toggle('activeline2__bars-menu');
+  lBars3.classList.toggle('activeline3__bars-menu');
+  listBars.toggle('deploy');
+});
+document.querySelector('.a2').addEventListener('click', () => {
+  lBars1.classList.toggle('activeline1__bars-menu');
+  lBars2.classList.toggle('activeline2__bars-menu');
+  lBars3.classList.toggle('activeline3__bars-menu');
+  listBars.toggle('deploy');
+});
```

Comment:
- [ ] Fix The issue and make sure that you project has fulfilled the following ...

- When the user clicks (or taps) the hamburger button, the mobile menu appears.
- When the user clicks (or taps) the close (X) button, the mobile menu disappear.
- When the user clicks (or taps) any of the mobile menu options, a correct part of the page is displayed.
- When the user clicks (or taps) any option of the mobile menu, the mobile menu disappear.
- The hamburger button and menu follows the design from [the chosen template in Figma]